.github/workflows/notify-about-new-emojis-material-icons.yml

```diff
@@ -39,6 +39,7 @@ jobs:
 
       - name: Check new Material Icons
         id: check-material-icons
+        continue-on-error: true
         run: python scripts/check_new_material_symbols.py
```

Comment:
We need `continue-on-error: true` here to trigger the step after sending a Slack notification when the checking new material icons step fails (this is expected when Google actually updates the material symbol font).